src/screens/Hashtag.tsx

```diff
@@ -79,7 +82,7 @@ export default function HashtagScreen({
     <View style={a.flex_1}>
       <ViewHeader
         title={headerTitle}
-        subtitle={author ? `By ${author}` : undefined}
+        subtitle={author ? `${_(msg`By`)} ${author}` : undefined}
```

Comment:
Wonder if the interpolated flavor would be easier for translators to grok instead of a lone word 🤔

Reply:
Wait what's the syntax on that?